src/entercommand.js

```diff
@@ -19,9 +19,9 @@ export default class EnterCommand extends Command {
 	/**
 	 * @inheritDoc
```

Comment:
We need to document this new option – just like here: https://github.com/ckeditor/ckeditor5-heading/blob/master/src/headingcommand.js#L63.